alica_standard_library/include/Conditions.h

```diff
@@ -14,6 +14,18 @@ bool AllChildFailure(const alica::Blackboard* input, const alica::RunningPlan* r
 bool AlwaysTrueCondition(const alica::Blackboard* input, const alica::RunningPlan* rp, const alica::Blackboard* globalBlackboard);
 bool AlwaysFalseCondition(const alica::Blackboard* input, const alica::RunningPlan* rp, const alica::Blackboard* globalBlackboard);
 bool IsChildSuccess(const alica::Blackboard* input, const alica::RunningPlan* rp, const alica::Blackboard* globalBlackboard);
+template <typename T>
+bool IsEqual(const alica::Blackboard* input, const alica::RunningPlan* rp, const alica::Blackboard* gb);
+template <typename T>
+bool IsNotEqual(const alica::Blackboard* input, const alica::RunningPlan* rp, const alica::Blackboard* gb);
+template <typename T>
+bool IsGreaterThan(const alica::Blackboard* input, const alica::RunningPlan* rp, const alica::Blackboard* gb);
+template <typename T>
+bool IsLessThan(const alica::Blackboard* input, const alica::RunningPlan* rp, const alica::Blackboard* gb);
+template <typename T>
+bool IsLessThanOrEqual(const alica::Blackboard* input, const alica::RunningPlan* rp, const alica::Blackboard* gb);
+template <typename T>
+bool IsGreaterThanOrEqual(const alica::Blackboard* input, const alica::RunningPlan* rp, const alica::Blackboard* gb);
```

Comment:
I thought the definition for template functions needs to be in the header

Reply:
This way it is working